docs/src/main/sphinx/admin/openmetrics.md

```diff
@@ -0,0 +1,134 @@
+# Cloud-scale Trino metrics with OpenMetrics
+
+Trino supports the metrics standard [OpenMetrics](https://openmetrics.io/), that
+originated with the open-source systems monitoring and alerting toolkit
+[Prometheus](https://prometheus.io/).
+
+Metrics are automatically enabled and available on the coordinator at the
+`/metrics` endpoint. The endpoint is protected with the configured
+[authentication](security-authentication), identical to the
+[](/admin/web-interface) and the [](/develop/client-protocol).  
+
+For example, you can retrieve metrics data from an unsecured Trino server
+running on `localhost:8080` with random username `example`:
+
+```shell
+curl -H X-Trino-User:foo localhost:8080/metrics
+```
```

Comment:
Because this works for any cluster the localhost should not be necessary (most users will never run trino on their PC)

it's necessary to point out users need system-information read permission

Its necessary to point out for complete metrics all workers will need to be scraped.